docs/source/runway_config.rst

```diff
@@ -30,6 +30,45 @@ Module
 
 .. autoclass:: runway.config.ModuleDefinition
 
+Path
+^^^^
+
+.. autoclass:: runway.path.Path
```

Comment:
Since `Path` is not being defined as a mapping in the config file, I don't see an advantage to including the kwargs on this doc page. If you use `.. automodule:: runway.path.Path` it will exclude the `__init__` docstring with the kwargs.

Reply:
Agreed, updating.